remedy.lic

```diff
@@ -125,11 +127,11 @@ class Remedy
     when 'You toss the'
       stow_item(left_hand)
       get_item('pestle')
-      crush("crush my #{@noun} with my pestle")
+      command = "crush my #{@noun} with my pestle"
     when 'You slowly filter', 'You spin your'
```

Comment:
Could you combine cases that have the same code with this syntax? `when 'A', 'B', 'C'`